common/lib/dependabot/metadata_finders/base/release_finder.rb

```diff
@@ -20,9 +20,16 @@ def initialize(source:, dependency:, credentials:)
         def releases_url
           return unless source
 
+          # Azure does not provide tags via API, so we can't check whether
+          # there are any releases. So, optimistically return the tags location
+          return "#{source.url}/tags" if source.provider == "azure"
```

Comment:
Getting the specs to pass revealed that Azure does not have a releases API that I could mock, so I couldn't get the existing test to pass.

Instead, I kept the existing behavior, which I think it makes sense. Since Azure does not have a releases API, we cannot check in advance whether there are any releases available, so optimistically including the link sounds reasonable to me.